GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -996,24 +997,19 @@ - (void)assertValidPresentingViewController {
 }
 
 #if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
-// Asserts the requested scopes are valid.
 - (void)assertValidScopes:(NSArray<NSString *> *)scopes {
-  NSDictionary *scopeToClassMapping = @{
-    kAccountDetailTypeAgeOver18Scope : [GIDVerifyAccountDetail class],
-  };
-  NSMutableString *errorMessage =
+  GIDRestrictedScopesRegistry *registry = [[GIDRestrictedScopesRegistry alloc] init];
+  NSDictionary<NSString *, Class> *restrictedScopesMapping = [registry restrictedScopeToClassMappingInSet:[NSSet setWithArray:scopes]];
```

Comment:
Is this line too long? If so, please wrap following our style.

Reply:
Good catch! Formatted.